app/controllers/tasks_controller.rb

```diff
@@ -0,0 +1,60 @@
+class TasksController < ApplicationController
+  def index
+    @tasks = Task.order(id: :asc)
+  end
+
+  def by_task
+   @task = Task.where(id: params[:id])
+   render :show
+  end
+
+  def new
+    @task = Task.new
+  end
+
+  def create
+    @task = Task.new(task_create_params[:task])
+    if (@task.save)
+      redirect_to root_path
+    else
+      render :new
+    end
+  end
+
+  def task_create_params
+    params.permit(task: [:name, :description, :person_id])
+  end
+
+  def task_edit_params
+    params.permit(task: [:name, :description, :person_id])
+  end
```

Comment:
Since task_create_params and task_edit_params are identical methods, you could just have one method that you call for creating and editing.